src/hb-ot-maxp-table.hh

```diff
@@ -48,6 +48,11 @@ struct maxp
     return numGlyphs;
   }
 
+  inline void set_num_glyphs (uint16_t count)
```

Comment:
Typically we just use `unsigned int` for these kinds of numbers, instead of narrower ones. If you want to be pedantic, you can set, then assert that the set value is equal to count. Eg.:
```
  numGlyphs.set (count);
  assert (numGlyphs == count);
```
That catches overflows.

Reply:
updated to unsigned int